### ISSUE TITLE
NVDA 2023.1 breaks addons again

### ISSUE THREAD

**Issue by KevanGP** [NVDARemote/NVDARemote#323]
Can NVDARemote be updated for latest NVDA please?

**Comment by tech10** [comment]
I thought #311 resolved this problem. Have you built the add-on from the latest master branch and tested it? I do know that there isn't an official release of the add-on for 2023.1 compatibility yet. However, there is also version 2.6 available [here](https://github.com/NVDARemote/NVDARemote/releases/tag/v2.6) that has already merged the PR mentioned above. Have you tested this, does it work for you?

**Comment by ctoth** [comment]
NVDA Remote was updated to work with the new API three weeks ago. So, no. It cannot be updated, as it is already updated.
